services/streammanager/kafka/client/producer.go

```diff
@@ -61,13 +65,15 @@ func (c *Client) NewProducer(topic string, producerConf ProducerConfig) (p *Prod
 	}
 
 	p = &Producer{
+		config: producerConf,
 		writer: &kafka.Writer{
 			Addr:                   c,
 			Topic:                  topic,
 			Balancer:               &kafka.ReferenceHash{}, // TODO replace: https://github.com/segmentio/kafka-go/pull/906
 			BatchTimeout:           time.Nanosecond,
 			WriteTimeout:           producerConf.WriteTimeout,
 			ReadTimeout:            producerConf.ReadTimeout,
+			MaxAttempts:            1,
```

Comment:
Although previously we were [using 3](https://github.com/Shopify/sarama/blob/main/config.go#L478), I am fine with 1 as well!